cmake/legacy.cmake

```diff
@@ -123,31 +123,31 @@ set(boost_features
 )
 ExternalProject_Add(boost
   URL "https://boostorg.jfrog.io/artifactory/main/release/1.${boost_version}.0/source/boost_1_${boost_version}_0.tar.bz2"
-  URL_HASH SHA256=1e19565d82e43bc59209a168f5ac899d3ba471d55c7610c677d4ccf2c9c500c0
+  URL_HASH SHA256=6478edfe2f3305127cffe8caf73ea0176c53769f4bf1585be237eb30798c3b8e
   BUILD_IN_SOURCE ON
   PATCH_COMMAND ${patch} -p1 -i "${CMAKE_SOURCE_DIR}/legacy/boost/0004-filesystem-posix-fix-no-at-apis-missing-include.patch"
```

Comment:
Have you checked that this patch is still needed?

Reply:
I did not check carefully but I can't remember that I have seen a reject in the log file. I will crosscheck again.

The question is if we want to go to the latests boost since this could add problems with the findboost module of cmake!!

Reply:
I removed the patch since the code is already in. I overlooked the output before.